### ISSUE TITLE
Modifications in ACGAN

### ISSUE THREAD

**Issue by rangwani-harsh** [POSTECH-CVLab/PyTorch-StudioGAN#74]
Hi,
Thanks for your awesome work!

Can you please specify the modifications which have been made in ACGAN-Mod which increase the performance of ACGAN?
Also can you please let us know how much is improvement that you get with those modifications?

Thanks
Harsh

**Comment by MINGUKKANG** [comment]
Hi,

I am sorry for the late reply. 

I have found that the ACGAN configuration specified in StudioGAN is not a setup for ACGAN-mod, which used in ContraGAN paper. 
(I forgot to update the configuration... sry)

Through exhaustive experiments, we have found that the following changes will enable ACGAN to generate more realistic images:

1) remove conditioning loss on fake images when training the discriminator
```python
original: dis_acml_loss += (self.ce_loss(cls_out_real, real_labels) + self.ce_loss(cls_out_fake, fake_labels))
modification: dis_acml_loss += self.ce_loss(cls_out_real, real_labels) 
```

2) apply ema update 

3) use the same hyperparameter setup used in ContraGAN

4) use hinge loss

4) apply spectral normalization both on the generator and discriminator

Since we do not record how the above heuristics improves generation performance of ACGAN, we can not tell you the effectiveness of the above remedies. 

But, I remembered that the above changes enable ACGAN to generate comparable generation outputs with ContraGAN and BigGAN on CIFAR10.

Thank you.

Best, 

Minguk
